xaitk_saliency/utils/bin/drise_on_img_dir.py

```diff
@@ -0,0 +1,61 @@
+import click  # type: ignore
+import os
+from PIL import Image  # type: ignore
+import json
+import matplotlib.pyplot as plt  # type: ignore
+import numpy as np
+
+from xaitk_saliency.utils.tools.drise_from_config import drise_from_config
+
+
+@click.command()
+@click.argument('img_dir')
+@click.argument('output_dir')
+@click.argument('config_file')
+@click.option('--overlay-image/--no-image', default=False)
+@click.option('--perturbations', default=100)
+@click.option('--verbose', '-v', count=True)
```

Comment:
We should have an option to output a generated config file with defaults and, more interestingly, possibly implementation sub-configs for the available plugins (right now for the `ImageObjectDetector` impls available, but also in the future the different proposal generation methods as well).